lib/requests/foods.js

```diff
@@ -79,21 +79,23 @@ const renderFood = (food) => {
   )
 }
 
-const updateFood = (id) => {
+const updateFood = (id, originalFood) => {
   var foodName = $(`.food-item-${id}`).children()[0].innerText
   var foodCalories = $(`.food-item-${id}`).children()[1].innerText
-  fetch(`https://ivmh-qs-api.herokuapp.com/api/v1/foods/${id}`, {
-    method: 'PUT',
-    headers: {'Content-Type': 'application/json'},
-    body: JSON.stringify({
-      food: {
-        name: foodName,
-        calories: foodCalories
-      }
+  if (originalFood.name != foodName || originalFood.calories != foodCalories ) {
```

Comment:
Perhaps we could pull out the functionality where the original value is checked against the value when focused out into another function - and only call updateFood when the food name/calorie value has actually changed.